server/src/main/java/com/datastrato/gravitino/server/web/rest/ExceptionHandlers.java

```diff
@@ -79,6 +79,16 @@ public static Response handleTopicException(
     return TopicExceptionHandler.INSTANCE.handle(op, topic, schema, e);
   }
 
+  public static Response handleUserPermissionOperationException(
+      OperationType op, String role, String parent, Exception e) {
```

Comment:
Here the parameter name should be `roles`, not `role`, right?

Reply:
Done.